### ISSUE TITLE
Screen reader cannot access Sounds description in Audio preferences

### ISSUE THREAD

**Issue by vperezcadavid99** [phetsims/vector-addition#440]
**Test device**
Asus Zenbook

**Operating System**
Win 11

**Browser**
Firefox+NVDA

**Problem description**
From https://github.com/phetsims/qa/issues/1336

Unable to hear the sound description: _"Sounds: Play sonifications and sound effects as you interact."_ from the Audio preferences menu with the screen reader.

**Steps to reproduce**
1. Open the Preferences menu.
2. Turn on the screen reader.
3. Using tab, navigate to Audio preferences.
4. Press the down key to hear the screen reader description. Note that there is no reading for this

**Note:**I just realized that this also happened for Graphing Quadratics

**Visuals**

https://github.com/user-attachments/assets/fc834016-53c3-48a3-beea-cb870aab3930

Troubleshooting information:
!!!!! DO NOT EDIT !!!!!
Name: ‪Vector Addition: Equations‬
URL: https://phet-dev.colorado.edu/html/vector-addition-equations/1.2.0-rc.1/phet/vector-addition-equations_all_phet.html
Version: 1.2.0-rc.1 2025-12-05 18:47:46 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:145.0) Gecko/20100101 Firefox/145.0
Language: es-ES
Window: 1440x768
Pixel Ratio: 2/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (ANGLE (AMD, Radeon HD 3200 Graphics Direct3D11 vs_5_0 ps_5_0), or similar)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

**Comment by jessegreenberg** [comment]
Thanks @vperezcadavid99 . This was also reported in https://github.com/phetsims/joist/issues/1034 but we never fixed it. I added the appropriate label to make sure it is on our prioritized board.

We will try to fix that issue soon, but I also don't think it needs to be fixed before releasing this sim.

**Comment by pixelzoom** [comment]
Thanks for investigating @jessegreenberg.  I agree that this does not need to be fixed for the 1.2 release.  So closing this issue as "won't fix".
